Compilation fails if libdcgm.so.4.1.1 is before libdcgmi_objects.a in the
generated command-line.
--
diff --git a/dcgmi/CMakeLists.txt b/dcgmi/CMakeLists.txt
index 0bb55e5..fa0c5fc 100755
--- a/dcgmi/CMakeLists.txt
+++ b/dcgmi/CMakeLists.txt
@@ -58,11 +58,11 @@ target_link_libraries(dcgmi
     PRIVATE
         buildinfo_objects
         common_interface
-        dcgm
         dcgm_interface
         dcgm_mutex
         dcgm_thread
         dcgmi_objects
+        dcgm
         fmt::fmt
         nvvs_interface)
